arctic/store/version_store.py

```diff
@@ -66,6 +66,10 @@ def _ensure_index(self):
         collection.versions.create_index([('symbol', pymongo.ASCENDING), ('version', pymongo.DESCENDING)], unique=True,
```

Comment:
@jamesblackburn , James are you happy with the changes, merge OK  ?  cc: @kraphtuos